examples/mnist/mnist.py

```diff
@@ -58,32 +59,47 @@ def run(train_batch_size, val_batch_size, epochs, lr, momentum, log_interval):
                                                      'nll': Loss(F.nll_loss)},
                                             device=device)
 
+    desc = "ITERATION - loss: {:.2f}"
+    pdbar = tqdm(
+        initial=log_interval, leave=False, total=len(train_loader),
```

Comment:
Shouldn't we this `initial=log_interval` and start from 0 ?

Reply:
Yes, you are right.  I have set the initial value to zero.